File: RP6/d7-2.java
import java.util.Scanner;

public class Ex7_V2 {

    public static String classificar_nota(double nota) {
        if (nota >= 90) {
            return "A";
        } else if (nota >= 80) {
            return "B";
        } else if (nota >= 70) {
            return "C";
        } else if (nota >= 60) {
            return "D";
        } else {
            return "E";
        }
    }

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        double nota;
        System.out.print("Digite a nota (0 a 100): ");
        nota = sc.nextDouble();

        String resultado = classificar_nota(nota);

        System.out.println("Classificação: " + resultado);
    }
}

File: RP6/d7-2.py
def classificar_nota(nota):
    if nota >= 90:
        return "A"
    elif nota >= 80:
        return "B"
    elif nota >= 70:
        return "C"
    elif nota >= 60:
        return "D"
    else:
        return "E"

nota = float(input("Digite a nota (0 a 100): "))
resultado = classificar_nota(nota)

print("Classificação:", resultado)